src/components/ActionMenu/MenuItems/AddNew.js

```diff
@@ -0,0 +1,41 @@
+import React, {
+  memo,
+} from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Icon, Button } from '@folio/stripes/components';
+
+import sharedCss from '../../../shared.css';
+import { createLayerURL } from '../../../utils';
+import { LAYER_TYPES } from '../../../utils/constants';
+
+export const AddNew = memo(props => {
+  const {
+    caption,
+    menu,
+    location,
+  } = props;
+
+  return (
+    <Button
+      data-test-new-item-menu-button
+      to={createLayerURL(location, LAYER_TYPES.CREATE)}
+      buttonStyle="dropdownItem"
+      buttonClass={sharedCss.linkButton}
+      onClick={menu.onToggle}
+    >
+      <Icon icon="plus-sign">
+        <FormattedMessage id={caption} />
+      </Icon>
+    </Button>
+  );
+});
+
+AddNew.propTypes = {
+  caption: PropTypes.string.isRequired,
+  menu: PropTypes.shape({}).isRequired,
```

Comment:
When you do not need to specify the nesting shape just write

`PropTypes.shape(object).isRequired`

Reply:
`PropTypes.object.isRequired`